source/trackers.json

```diff
@@ -140,6 +140,12 @@
             "url": "https://branch.io/",
             "companyId": "branch_metrics_inc"
         },
+        "braze": {
```

Comment:
already in database
could be simply change the company name and add a tracker that is not already in the database 
`"braze.com": "braze",`